doc/sphinx/source/analysis.txt

```diff
@@ -0,0 +1,34 @@
+=================
+ Analysis modules
+=================
+
+.. versionadded:: 0.8.0
+The Analysis modules help in the implementation analyses of
+MDPOW simulations. To simplify the process of analyzing
+a collection of systems generated by a free energy simulation
+the objects in mdpow.ensemble allow for a molecule directory's
+systems to be loaded into `MDAnalysis <https://mdanalysis.org>`_
+Universes and be analyzed as a group.
+
```

Comment:
Add a toc tree here, like
```reST
.. toctree::
   :maxdepth: 1
   :numbered:
   :hidden:

   analysis/ensemble
```
and then other txt files inside `analysis/` can be added later.

Reply:
done